assets/js/googlesitekit/components-gm2/ProgressBar.js

```diff
@@ -21,17 +21,50 @@
  */
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import {
+	BREAKPOINT_DESKTOP,
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	BREAKPOINT_XLARGE,
+	useBreakpoint,
+} from '../../hooks/useBreakpoint';
 
 export default function ProgressBar( {
 	className,
 	small,
 	compress,
 	indeterminate,
 	height,
+	smallHeight,
+	tabletHeight,
+	desktopHeight,
 	progress,
 } ) {
+	const breakpoint = useBreakpoint();
+
+	let progressBarHeight = height;
+
+	if ( BREAKPOINT_SMALL === breakpoint && smallHeight ) {
```

Comment:
I know it's unlikely that `smallHeight` etc will ever have the value `0` but nonetheless it would be safer here to check for `smallHeight !== undefined` (and so on, below).